nixos/modules/system/boot/loader/generations-dir/generations-dir-builder.sh

```diff
@@ -51,36 +51,42 @@ addEntry() {
     local path="$1"
     local generation="$2"
     local outdir=/boot/system-$generation
+    local defaultdir=/boot/default
 
     if ! test -e $path/kernel -a -e $path/initrd; then
         return
     fi
 
     local kernel=$(readlink -f $path/kernel)
     local initrd=$(readlink -f $path/initrd)
+    local relative=""
 
     if test -n "@copyKernels@"; then
         copyToKernelsDir $kernel; kernel=$result
         copyToKernelsDir $initrd; initrd=$result
+        relative="r"
```

Comment:
This is a bit ugly. Why not name it like extraLnArgs or something like that? or use a boolean type instead?

Reply:
Cleaned this up.